llvm/lib/Target/RISCV/RISCVTargetTransformInfo.cpp

```diff
@@ -1534,6 +1534,13 @@ RISCVTTIImpl::getArithmeticReductionCost(unsigned Opcode, VectorType *Ty,
              getRISCVInstructionCost(Opcodes, LT.second, CostKind) +
              getCmpSelInstrCost(Instruction::ICmp, ElementTy, ElementTy,
                                 CmpInst::ICMP_EQ, CostKind);
+    } else if (ISD == ISD::XOR) {
+      // Example sequences:
+      //   vsetvli a0, zero, e8, mf8, ta, ma
+      //   vcpop.m a0, v0
+      //   andi a0, a0, 1
```

Comment:
Doesn't the else block already cover this case?

Reply:
Yes， fixed

Reply:
XOR doesn't use a seqz though so it shouldn't use `getCmpSelInstrCost`?

Reply:
@lukel97 You're right. I initially considered replacing getCmpSelInstrCost with 1 to streamline the code. However, I'm fine if you'd prefer to handle the cost separately.

Reply:
fixed